UI/DialogSystem/DialogSystem.cs

```diff
@@ -0,0 +1,71 @@
+using Godot;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public class DialogSystem : CanvasLayer 
```

Comment:
xmldoc to explain how to use this dialog system class would be cool

Reply:
Will come when its ready for use.